crates/agent_ui/src/active_thread.rs

```diff
@@ -2025,152 +2028,162 @@ impl ActiveThread {
             }
         });
 
-        let styled_message = match message.role {
-            Role::User => v_flex()
-                .id(("message-container", ix))
-                .pt_2()
-                .pl_2()
-                .pr_2p5()
-                .pb_4()
-                .child(
+        let styled_message = if message.ui_only {
+            self.render_ui_notification(message_content, ix, cx)
```

Comment:
This new conditional is the only change here; the rest is just indentation because the `match` is now surrounded by an `if`.